### DISCUSSION TITLE
Filter Bar behavior

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#4111] [Help: Figma] [answered]
Hi - I noticed when dragging in the Filter Bar component that the buttons on the right don't layout horizontally like they're intended to:

![Screenshot 2024-10-17 at 10 06 31 AM](https://github.com/user-attachments/assets/4c0d9f34-402c-4f1c-b5a5-82a16ceade9a)

I can't seem to fix it by resizing, so I wanted to bring it up in case it's a bug? Thanks!

**Comment by serifluous** [comment, marked as answer]
Ooh, good catch! Just published an update to fix this.

**Reply by ceyerkes** [reply]
thank youuu!
